### PR TITLE
fix: added the missing ids in parameterized tests

### LINKED ISSUE #362: Missing ids for parametrized tests
### Is your feature request related to a problem?

Some parametrized tests don't have `ids`.

### Desired solution

Add ids for the parametrized tests if missing

### Possible alternatives (optional)

_No response_

### Screenshots (optional)

_No response_

### Additional Context (optional)

_No response_

### PR DESCRIPTION
**Pull request by PhilipGutberlet**
Closes #362 

### Summary of Changes

Some parameterized tests didn't have ids. The missing ids are added now.

### CODE REVIEW

File: tests/safeds/data/tabular/transformation/test_standard_scaler.py [hunk 1]
```diff
@@ -143,6 +143,7 @@ class TestFitAndTransformOnMultipleTables:
                 ),
             ),
         ],
+        ids=["two columns"],
```
Comment (jxnior01) [inline tests/safeds/data/tabular/transformation/test_standard_scaler.py:146]:
```suggestion
        ids=["two_columns"],
```

File: tests/safeds/data/tabular/transformation/test_standard_scaler.py [hunk 2]
```diff
@@ -224,6 +225,7 @@ class TestInverseTransform:
                 },
             ),
         ],
+        ids=["one column"],
```
Comment (jxnior01) [inline tests/safeds/data/tabular/transformation/test_standard_scaler.py:228]:
```suggestion
        ids=["one_column"],
```

**Comment by daniaHu** [comment]
There are a couple functions, where ids are missing in test_image.py

**Comment by Marsmaennchen221** [comment]
@PhilipGutberlet please add the "Time spent by team" for your Issue

**Review by Marsmaennchen221 — CHANGES_REQUESTED** [review]
Looks good so far.

You missed the following tests:
`test_should_raise_if_index_out_of_bounds` in _table/test_slice_rows.py
`test_should_return_sorted_table` in _table/test_sort_rows.py
`test_should_not_modify_original_table` in _table/test_sort_rows.py
`test_should_return_dict_for_table` in _table/test_to_dict.py
`test_should_raise_if_column_not_found` in _table/test_transform_table.py

You also missed tests in the tests for Image

**Review by Marsmaennchen221 — DISMISSED** [review]

**Review by Marsmaennchen221 — APPROVED** [review]

**Pull request merged**